BenchmarkDotNet.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2027
+# Visual Studio Version 17
```

Comment:
We avoid updating the VS version as we don't want to force the contributors to use latest VS, however they need to install .NET 7 SDK anyway. Is this version the oldest that supports .NET 7?

Reply:
That was an automatic change by VS when I moved the project out of the sln. But yes, that also happens to be the oldest VS that supports net7.